openmls/src/extensions/mod.rs

```diff
@@ -85,30 +88,37 @@ pub enum ExtensionType {
     /// Reserved. This must not be used.
     Reserved = 0,
 
+    /// The application id extension allows applications to add an explicit,
+    /// application-defined identifier to a KeyPackage.
+    ApplicationId = 1,
+
+    /// The ratchet tree extensions provides the whole public state of the ratchet
+    /// tree.
+    RatchetTree = 2,
+
+    /// The required capabilities extension defines the configuration of a group
+    /// that imposes certain requirements on clients in the group.
+    RequiredCapabilities = 3,
+
+    /// To join a group via an External Commit, a new member needs a GroupInfo
+    /// with an ExternalPub extension present in its extensions field.
+    ExternalPub = 4,
```

Comment:
Can you add external sender while you're at it? Then we have all values from the RFC.